### DISCUSSION TITLE
Piped Everything: Config additions

### DISCUSSION THREAD

**Discussion by ECHOSUD** [Truinto/ONI-Modloader-SimpleMods#62] [General]
Steam workshop comments are limited to 1000 characters. I'm presuming then this is the best place to host config additions. If not intended to be apart of the base mod, it would be nice if the description would reflect where users can go find community made configs.

# Config Additions #
## Refrigerator & Big Storage: Restoraged ##
Using the same config options as StorageLocker, the following should be the config for Vanilla Refrigerator and the buildings from the [Big Storage: Restoraged](https://steamcommunity.com/sharedfiles/filedetails/?id=3059711743) mod.
Because the config is the same as StorageLocker, I haven't made the filters (Refrigerator & BigRefrigerator) or storage capacity more specific.
<details><summary><h3>config</h3></summary><pre>
		{
			"Id": "Refrigerator",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Solid"
			],
			"StorageCapacity": "Infinity"
		},
		{
			"Id": "BigRefrigerator",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Solid"
			],
			"StorageCapacity": "Infinity"
		},
		{
			"Id": "BigBeautifulStorageLocker",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Solid"
			],
			"StorageCapacity": "Infinity"
		},
		{
			"Id": "BigSmartStorageLocker",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Solid"
			],
			"StorageCapacity": "Infinity"
		},
		{
			"Id": "BigStorageLocker",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Solid"
			],
			"StorageCapacity": "Infinity"
		},
		{
			"Id": "BigStorageTile",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Solid"
			],
			"StorageCapacity": "Infinity"
		},
		{
			"Id": "StorageTile",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Solid"
			],
			"StorageCapacity": "Infinity"
		}</pre></details>

**Comment by ECHOSUD** [comment]
## Deodorizer, Oxygen Diffuser,  Rust Deoxidizer ##
- Deodorizer 
  - Output: Clay 
  - Input: NONE (1x1 building size) 
- Oxygen Diffuser 
  - Input: Algae 
- Rust Deoxidizer 
  - Input: Rust, Salt 
  - Output: Iron Ore

<details><summary><h3>config</h3></summary><pre>
{
			"Id": "AirFilter",
			"Input": false,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Clay"
			]
		},
		{
			"Id": "MineralDeoxidizer",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Algae"
			],
			"StorageCapacity": 330
		},
		{
			"Id": "RustDeoxidizer",
			"Input": true,
			"OffsetX": 0,
			"OffsetY": 0,
			"Filter": [
				"Rust",
				"Salt"
			],
			"StorageCapacity": 780
		},
		{
			"Id": "RustDeoxidizer",
			"Input": false,
			"OffsetX": 1,
			"OffsetY": 0,
			"Filter": [
				"IronOre"
			]
		}</pre></details>

**Comment by Truinto** [comment]
Steam has a discussion tab for each object. I think that's a fine place. Otherwise github also has a discussion tab. It will be lost quickly in the "issues" tab. You can handle it as you like. There is very little interest in modifications. Most people will just uninstall, if it does not do what they want.

**Comment by VoidedWinter** [comment]
Looking for output for the dehydrator.  Can't stand that the water just dumps on the floor.

**Comment by GhChancellor** [comment]
I added these:

    {
      "Id": "AirFilter",
      "Input": false,
      "OffsetX": 0,
      "OffsetY": 0,
      "Filter": [
        "Oxygen"
      ],
      "Color": {
        "r": 255,
        "g": 255,
        "b": 255,
        "a": 255
      }
    },
    {
      "Id": "CO2Scrubber",
      "Input": true,
      "OffsetX": 1,
      "OffsetY": 1,
      "Filter": [
        "CarbonDioxide"
      ],
      "Color": {
        "r": 255,
        "g": 255,
        "b": 255,
        "a": 255
      }
    },

and i changed these:

{
      "Id": "Generator",
      "Input": false,
      "OffsetX": 1,
      "OffsetY": 2,
      "Filter": [
        "CarbonDioxide"
      ],
      "Color": {
        "r": 50,
        "g": 205,
        "b": 50,
        "a": 255
      }
    },
    {
      "Id": "WoodGasGenerator",
      "Input": false,
      "OffsetX": 0,
      "OffsetY": 1,
      "Filter": [
        "CarbonDioxide"
      ],
      "Color": {
        "r": 50,
        "g": 205,
        "b": 50,
        "a": 255
      }
    },

bye

**Comment by Methodiuzz** [comment]
I've added the water cooler
`{
      "Id": "WaterCooler",
      "Input": true,
      "OffsetX": 0,
      "OffsetY": 0,
      "Filter": [
        "Water"
      ]
    }`
